### PR TITLE
added pdf archiving functionality

### PR DESCRIPTION
**Pull request by voisardf**
Generalized functionality to archive the generated PDF in a configurable folder. If non is given, the PDF's are not duplicated for archiving

### CODE REVIEW

File: crdppf/views/printproxy.py [hunk 1]
```diff
@@ -113,9 +113,21 @@ def status(self):
...
+
+        pdf = self._proxy_response(
             "%s/report/%s" % (
                 self.config['print_url'],
                 self.request.matchdict.get('ref')
             ),
         )
+        
+        if archive_path is not None:
+            filename = pdf.content_disposition.split('=')[1]
```
Comment (voisardf) [inline crdppf/views/printproxy.py]: or is there a better way to get the filename I didn't see?

Comment (kalbermattenm) [reply crdppf/views/printproxy.py]: Uh... I do not like it, but I cannot figure out another way.

What does the PDF name look like ? Would it be possible to use something which comes from `self.request.params` ?

File: crdppf/views/printproxy.py [hunk 2]
```diff
@@ -113,9 +113,21 @@ def status(self):
...
+            with open(archive_path+filename, 'wb') as f:
```
Comment (kalbermattenm) [inline crdppf/views/printproxy.py]:
```python
import os
...
with open(os.path.join(archive_path, filename), 'wb') as f:
...
```

File: crdppf/views/printproxy.py (hunk 1 above)
Comment (voisardf) [reply crdppf/views/printproxy.py]: Probably we could, but we want to keep the initial creation date. I might improve that later on

**Comment by voisardf** [comment]
Ammended following your comments. Merging

**Pull request merged**